### DISCUSSION TITLE
Clearful

### DISCUSSION THREAD

**Discussion by Jethro87** [ABausG/home_widget#290] [Show and tell]
👋 We've been using home_widget in [Clearful](https://clearful.com/) for a couple of years and couldn't be happier. The communication between our app and our widgets is made very easy with home_widget, and the integration experience was straightforward. Thanks for this package, Anton, and for being an overall great guy!

<img width="1358" alt="clearful-home-widget" src="https://github.com/user-attachments/assets/9ddcc3d0-f639-4d82-9ceb-d68ef17f401d">

**Comment by Shabaquib** [comment]
Looks really amazing, i first was trying to learn to create widgets with Jetpack Compose but failed, but then searched if this functionality was available in Flutter and found that this packages uses Jetpack glance to work. Can't wait to build amazing stuffs with it!

**Comment by gauravghodinde** [comment]
Hey, these are some really good looking widgets, im trying do the same but im getting error while importing import es.antonborri.home_widget.HomeWidgetPlugin
can you suggest how to solve it
![image](https://github.com/user-attachments/assets/267960d6-0329-4064-8475-6bfee7077cc2)
